src/Extension/CoreExtension.php

```diff
@@ -920,6 +921,38 @@ public static function reverseFilter(Environment $env, $item, $preserveKeys = fa
         return $string;
     }
 
+    /**
+     * Shuffle an array, a \Traversable instance, or a string.
+     * The function does not preserve keys.
+     *
+     * @internal
+     */
+    public static function shuffleFilter(Environment $env, mixed $item): mixed
```

Comment:
```suggestion
    public static function shuffleFilter(string $charset $item): mixed
```

For `$item`, we can be more precise than `mixed` o the phpdocs.